docs/modules/commons/pages/variables.adoc

```diff
@@ -219,3 +219,30 @@ When the condition `#{eval(`${optionalPathToCheck:/defaultPath}` != `/forbiddenP
 |step                                                                |
 |When I go to the relative URL '${optionalPathToCheck:/defaultPath}' |
 ----
+
+=== Complex types variables
+
+Vividus provides the ability to work with not only primitive variables like numbers and strings but also supports complex types such as lists and maps.
```

Comment:
our users are not technical, they might not know data structures, thus we should avoid describing tables via lists and maps

Reply:
our users can use conditions and loops, evaluate expressions and JS scripts, but will they be frustrated by lists and maps?

Reply:
updated